packages/manager/apps/pci-quota/src/pages/quota/IncreaseQuotaSupport.page.tsx

```diff
@@ -149,28 +151,29 @@ const IncreaseQuotaSupport = () => {
 
   const { mutate: createTicket, isPending } = useMutationCreateTicket({
     onSuccess: (data: TCreateTicketResponse) => {
+      setTicketCreated(true);
+      const ticketUrl =
+        SUPPORT_TICKET_ID_URL.replace('{ticketId}', data.ticketId) +
+        me.ovhSubsidiary;
```

Comment:
For readability & consistency concerns, I'll suggest to use template literals instead :)